db/migrate/20240507004300_add_user_id_to_medical_records.rb

```diff
@@ -0,0 +1,11 @@
+class AddUserIdToMedicalRecords < ActiveRecord::Migration[7.1]
+  def change
+    user = User.first || User.create(email: "temporary_email@example.com", password: "pass", password_confirmation: "pass")
```

Comment:
Please ensure this line is working, I mean, maybe the password doesn't meet the requirements.